docs/user/tutorials/01-10-aws-nfs-volume.md

```diff
@@ -0,0 +1,188 @@
+# AWS NFS Volume
+
+This tutorial explains how to create and use RWX volume in AWS cloud provider that can be used from multiple workloads.
+
+## Steps <!-- {docsify-ignore} -->
+
+1. Create a namespace and export its value as an environment variable. Run:
+
+   ```shell
+   export NAMESPACE={NAMESPACE_NAME}
+   kubectl create ns $NAMESPACE
+   ```
+   
+2. Create AwsNfsVolume resource.
+
+   ```shell
+   cat <<EOF | kubectl -n $NAMESPACE apply -f -
+   apiVersion: cloud-resources.kyma-project.io/v1beta1
+   kind: AwsNfsVolume
+   metadata:
+     name: my-vol
+   spec:
+     capacity: 100G
+   EOF
+   ```
+   
+3. Wait for AwsNfsVolume to get Ready condition.
+
+   ```shell
+   kubectl -n $NAMESPACE wait --for=condition=Ready awsnfsvolume/my-vol --timeout=300s
+   ```
+
+   Once created AwsNfsVolume is provisioned this command should terminate with message:
+
+   ```
+   awsnfsvolume.cloud-resources.kyma-project.io/my-vol condition met
+   ```
+   
+4. Observe the generated PersistentVolume:
+   
+   ```shell
+   kubectl -n $NAMESPACE get persistentvolume my-vol
+   ```
+   
+   This command should print:
+   
+   ```
+   NAME     CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM            STORAGECLASS
+   my-vol   100G       RWX            Retain           Bound    test-mt/my-vol               
+   ```
+   
+   Note the RWX access mode which allows volume to be readable and writable from multiple workloads, and
+   Bound status which means PersistentVolumeClaim claiming this PV is created.
+   
+5. Observe the generated PersistentVolumeClaim:
+
+   ```shell
+   kubectl -n $NAMESPACE get persistentvolumeclaim my-vol
+   ```
+
+   This command should print:
+
+   ```
+   NAME     STATUS   VOLUME   CAPACITY   ACCESS MODES   STORAGECLASS 
+   my-vol   Bound    my-vol   100G       RWX                         
+   ```
+
+   Similarly to PV, note the RWX access mode and Bound status.
+
+6. Create two workloads that both will write to the volume
```

Comment:
```suggestion
6. Create two workloads that both write to the volume
```
What does it mean that the workloads write to the volume?

Reply:
Both workloads writing to volume means there are two pods with apps writing to the same file on that NFS disk / volume